src/components/encoding-pane/property-editor.tsx

```diff
@@ -14,24 +19,61 @@ export interface PropertyEditorProps extends ActionHandler<SpecEncodingAction> {
   fieldDef: ShelfFieldDef;
 }
 
+export interface PropertyEditorSchema {
+  uiSchema: any;
+  schema: any;
+}
+
 export class PropertyEditorBase extends React.PureComponent<PropertyEditorProps, {}> {
+  private static getPropertyEditorSchema(prop: string, nestedProp: string): PropertyEditorSchema {
```

Comment:
Do they use anything from this class, if not, better just declare in the schema file so you only export this function and everything else remain encapsulate in the file?